api/app/weather_models/precip_rdps_model.py

```diff
@@ -50,13 +83,14 @@ def get_raster_keys_to_diff(timestamp: datetime):
     # From earlier model run, get the keys for 24 hours before timestamp and the timestamp to perform the diff
     earlier_key = f"{key_prefix}/"
     later_key = f"{key_prefix}/"
-    later_key = later_key + compose_computed_precip_rdps_key(target_model_run_date, target_model_run_date.hour, target_model_run_date.hour + 24)
     if target_model_run_date.hour != 0 and target_model_run_date.hour != 12:
         # not a model run hour, return earlier and later keys to take difference
-        earlier_key = earlier_key + compose_computed_precip_rdps_key(target_model_run_date, target_model_run_date.hour, target_model_run_date.hour)
+        later_key = later_key + compose_computed_precip_rdps_key(target_model_run_date, target_model_run_date.hour, target_model_run_date.hour + 24, SourcePrefix.COMPUTED)
```

Comment:
I'm not clear on why we need the computed prefix here? I thought that anytime we need to subtract rasters we always use the CMC grib2 files as our starting point. I would expect us to only need the computed raster when we're looking for 24 hour precip to feed into a dc/dmc/hffmc calculation.

Reply:
If we use the same filename as the model then we overwrite model inputs with computed outputs, limiting our ability to audit our calculations. We could do this all in memory but I'd rather leave this as files to audit things at least initially.

Reply:
thanks updated in: https://github.com/bcgov/wps/pull/3762/commits/deda92f1f4a5f42ff8db40c5c89693109b8ce7b4

Reply:
Looks good!